NOTICE.txt

```diff
@@ -10907,6 +10907,36 @@ OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
+--------------------------------------------------------------------------------
+Dependency : github.com/blakerouse/service
```

Comment:
We should probably move this repo under elastic?

Reply:
At the moment this is just pointing to a fix of the upstream, once this lands in the upstream repo then the dependency will not point to my personal repo but instead it will point to: https://github.com/kardianos/service

Which is what is defined in the `go.mod`, its just that I am using `replace` at the moment to get the needed fix for systemd on link from my repository.

Reply:
Can you file an issue to switch it over when upstream is fixed. Otherwise we forget for sure ;-)